in-touch/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
```suggestion
xtest('renders learn react link', () => {
```
Aby był test nieaktywny